app/services/admin/collections_csv_generator.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Admin
+  # Converts a set of collections to CSV.
+  class CollectionsCsvGenerator
+    def self.generate(collections)
+      new(collections).generate
+    end
+
+    def initialize(collections)
+      @collections = collections
+    end
+
+    def generate
+      CSV.generate do |csv|
+        csv << headers
+        collections.each do |collection|
+          csv << row(collection)
+        end
+      end
+    end
+
+    private
+
+    attr_reader :collections
+
+    # rubocop:disable Metrics/MethodLength
+    def headers
+      [
+        'collection title',
+        'collection id',
+        'collection druid',
+        'state',
+        'version number',
+        'creator',
+        'managers',
+        'date created',
+        'date last modified',
+        'release setting',
+        'visibility setting',
+        'license',
+        'DOI yes/no',
+        'review workflow'
+      ]
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    def row(collection)
+      [
```

Comment:
It might be easier to read if you have:
```
collection_version = collection.head
```

here at the beginning, then use that for cols 0, 3, 4, 8